tests/test.robot

```diff
@@ -0,0 +1,77 @@
+*** Settings ***
+Library  MicrobitGPIO
+Library  MQTTLibrary
+Library  String
+
+Suite Setup  Connect to Devices
+
+*** Variables ***
+${MQTT Broker}  192.168.211.91
+${Microbit Port}  /dev/ttyACM0
+${Baud Rate}  115200
+${Analog Pin}  1
+${Topic}  Test 
+
+*** Test Cases ***
+Test Normal condition
+    Connect to Microbit
+    Connect to MQTT Broker
```

Comment:
We are already connecting to devices in "Suite Setup". Why are we connecting here?